components/bsx/Offer/OfferList.vue

```diff
@@ -10,6 +10,9 @@
       :accountId="accountId"
       :isOwner="isOwner" />
   </CollapseCardWrapper>
+  <CollapseCardWrapper :label="$t('nft.offer.label')" v-else>
+    <p>{{ 'There are currently no offers for this NFT.' }}</p>
```

Comment:
would it be possible to add this into en.json? key: `nft.offer.empty`

Reply:
added translation